module/os/linux/zfs/zvol_os.c

```diff
@@ -177,10 +311,11 @@ zvol_write(zv_request_t *zvr)
 
 	rw_exit(&zv->zv_suspend_lock);
 
-	if (acct)
+	if (!zv->zv_zso->use_blk_mq) {
```

Comment:
```suggestion
	if (!zv->zv_zso->use_blk_mq && acct) {
```
Since it's possible for `blk_queue_io_stat()` to return false above.